librz/util/list.c

```diff
@@ -627,3 +627,13 @@ RZ_API char *rz_list_to_str(RzList *list, char ch) {
 	}
 	return rz_strbuf_drain(buf);
 }
+
+RZ_API RzList *rz_list_of_sdblist(SdbList *sl) {
+	RzList *l = rz_list_newf(free);
```

Comment:
I think you need to cast to avoid warning?

Reply:
What warning?

Reply:
Ignore me :D